### DISCUSSION TITLE
Virtual iFit Not Working

### DISCUSSION THREAD

**Discussion by rjlee** [cagnulein/qdomyos-zwift#2464] [General]
Hi,

I've tried enabling the 'Virtual iFit' device, but didn't seem to have much success.  My setup is:

qz app running on Samsung Android Phone - v 2.16.55
iFit app running on Lenovo M11 Tab Tablet
NordicTrack S22i running companion app 

I've got power and cadence reporting fine in the qz app and zwift works fine with incline control.

When I try to launch a workout in iFit, the bike/qz pairs with iFit and then the workout works fine, but there are no changes to incline or resistance - I know the feature is experimental, but not sure what is supposed to work and what isn't.  Has anyone had this working at all ?

There is an issue here: https://github.com/cagnulein/qdomyos-zwift/issues/441 which I'm happy to contribute to, just not sure what the expected behaviour is ?

**Comment by cagnulein** [comment]
Hi, i'm currently on vacation anyway send me a debug log and i will check
it on monday

https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

Thanks

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 20 lug 2024 alle 20:53 Robert Lee ***@***.***>
ha scritto:

**Reply by rjlee** [reply]
Thanks, it's not urgent so no rush.  I'll raise an issue and add a debug log on Monday.
